x-pack/plugins/canvas/server/sample_data/ecommerce_saved_objects.json

```diff
@@ -4,10 +4,11 @@
     "type": "canvas-workpad",
     "updated_at": "2018-10-22T15:19:02.081Z",
     "version": 1,
-    "migrationVersion": {},
+    "migrationVersion": {
```

Comment:
Since I don't know what this is used for, I might as well ask... what is this used for?

Reply:
This is to indicate to the migrator what version the data is migrated up to. Having `{}` indicates that the system needs to run all the `canvas-workpad` migrations when creating the record. Having `7.0.0` value here indicates to the migrator that the record is migrated up to version 7.0.0.

In this scenario the migrator won't need to run any migrations since the `id` field is already removed.

Reply:
Ah, but it allows us to have, for example, an 8.0.0 migration in the future, and it would only run that since it knows it's already on 7.0.0. That's neat, thanks for the explanation!